client/ionic/src/components/Carousel.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  IonContent,
+  IonPage,
+  IonSlides,
+  IonSlide,
+  IonCardSubtitle,
+  IonCardTitle,
+  IonImg,
+} from '@ionic/react';
+import 'tachyons';
+import useDynamicFlow from '../hooks/useDynamicFlow';
+import TopNav from '../components/TopNav';
+import { TextImageScreen, LoadedFlow } from '../content/flow';
+
+const TextImage = ({
+  screen,
+  flow,
+}: {
+  screen: TextImageScreen;
+  flow: LoadedFlow;
+}) => {
+  return (
+    <IonSlide>
+      <div className="flex flex-column justify-around h-100 pa4">
+        <div className="mt3 tl">
+          <div style={{ fontSize: '1.5em' }}>{screen.headingText}</div>
```

Comment:
nit: recommend conditionally rendering this div based on whether `headingText` exists on the screen rather than leaving an empty div

Reply:
Done